packages/protocol/blockchain_scripts/airdrop/2022-01-24-backers/calculation.ts

```diff
@@ -26,55 +22,74 @@ interface RawCSV {
 
 interface PreparedRow {
   backerAddress: string
-  immediatelyAvailable: BigNum,
+  immediatelyAvailable: BigNum
   vesting: BigNum
 }
 
 function sanitizeData(data: RawCSV[]): PreparedRow[] {
-  const toBigInt = (s: string) => new BigNum(s.replace(/,/g, ""))
-    .multipliedBy(decimals.toString())
+  const toBigInt = (s: string) => new BigNum(s.replace(/,/g, "")).multipliedBy(decimals.toString())
 
-  return data.map(row => ({
-      backerAddress: row['Backer Address'],
-      immediatelyAvailable: toBigInt(row['Immeditely available']),
-      vesting: toBigInt(row['Unlocks over 12 months']),
+  return data.map((row) => ({
+    backerAddress: row["Backer Address"],
+    immediatelyAvailable: toBigInt(row["Immeditely available"]),
+    vesting: toBigInt(row["Unlocks over 12 months"]),
   }))
 }
 
 async function saveImmediateData(data: PreparedRow[], path: string) {
   console.log(`Saving ${data.length} rows for no-vesting data to ${path}`)
-  await fs.writeFile(path, JSON.stringify(data.map(row => ({
-    account: row.backerAddress,
-    reason: LIQUIDITY_PROVIDER_GRANT_REASON,
-    grant: {
-      amount: row.immediatelyAvailable.toFixed(),
-      cliffLength: "0",
-      vestingInterval: "0",
-      vestingLength: "0",
-    }
-  })), null, 2))
+  await fs.writeFile(
+    path,
+    JSON.stringify(
+      data.map((row) => ({
+        account: row.backerAddress,
+        reason: LIQUIDITY_PROVIDER_GRANT_REASON,
+        grant: {
+          amount: row.immediatelyAvailable.toFixed(),
+          cliffLength: "0",
+          vestingInterval: "0",
+          vestingLength: "0",
+        },
+      })),
+      null,
+      2
+    )
+  )
 }
 
 async function saveVestingData(data: PreparedRow[], path: string) {
   console.log(`Saving ${data.length} rows for no-vesting data to ${path}`)
-  await fs.writeFile(path, JSON.stringify(data.map(row => ({
-    account: row.backerAddress,
-    reason: LIQUIDITY_PROVIDER_GRANT_REASON,
-    grant: {
-      amount: row.vesting.toFixed(),
-      cliffLength: "0",
-      vestingInterval: "2628000",  //  1 month
-      vestingLength:  "31536000",  // 12 months
-    }
-  })), null, 2))
+  await fs.writeFile(
+    path,
+    JSON.stringify(
+      data.map((row) => ({
+        account: row.backerAddress,
+        reason: LIQUIDITY_PROVIDER_GRANT_REASON,
```

Comment:
the reason isn't the liquidity provider grant reason. It would be a new one, like "Backer Airdrop", I would think.